addons/Animation/src/presenter.js

```diff
@@ -724,6 +777,8 @@ function AddonAnimation_create (){
     };
 
     presenter.validateModel = function(model) {
+        presenter.setSpeechTexts(model['speechTexts']);
```

Comment:
validate model coś ustawia? Nie powinno tutaj być dodatkowych "side effect".

Reply:
Ta sama konwencja była stosowana wszędzie, od samego początku dodania TTSa. Można by co prawda dodawać speech texty jako część wyniku validateModel, żeby zostało dodane do konfiguracji, ale w tym momencie działa to inaczej niż wszędzie indziej. Rozumiem oczywiście o co ci chodzi, ale z dwojga złego wolałem trzymać się ustalonej konwencji.

Reply:
Ustaloną konwencją było trzymanie validateModel jako pure function. Dodanie tutaj side effectu zostało wprowadzone dopiero dzięki TTS, i powinno być to skutecznie eliminowane z tego kodu.